kin/blockchain/environment.py

```diff
@@ -0,0 +1,36 @@
+"""Contains the Environment class to configure horizon"""
+from hashlib import sha256
+
+from stellar_base.network import NETWORKS
+from stellar_base.asset import Asset
+
+from .utils import is_valid_address
+
+
+class Environment:
+    # Environments holds the parameters that will be used to connect to horizon
+    def __init__(self, name, horizon_endpoint_uri, network_passphrase, kin_issuer, friendbot_url = None):
+        """
+
+        :param str name: Name of the environment.
+        :param str horizon_uri: a Horizon endpoint.
+        :param str network_passphrase: The passphrase/network_id of the environment.
+        :param str kin_issuer: The issuer of the KIN asset.
+        :param str friendbot_url: a url to a friendbot service
```

Comment:
elaboration is needed here, what's exactly friendbot, and emphasize that it's relevant on test network only